icons/turtle.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4.7 8a8.26 8.26 0 0 0 10.6 0" />
+  <path d="M14 14a4 4 0 0 0-8 0v3c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1v-3a8 8 0 0 1 16 0v3c0 .6-.4 1-1 1h-2c-.6 0-1-.4-1-1Z" />
+  <path d="M17 10h3a2 2 0 0 1 0 4H2" />
```

Comment:
The shell feels a bit off when scaled up, how about?
![image](https://github.com/lucide-icons/lucide/assets/17746067/ce68ddb7-666d-4b49-92db-ca053571053c)

```suggestion
  <path d="M8 10h4" />
  <path d="m15 8-3 2 2 4v3c0 .6.4 1 1 1h2c.6 0 1-.4 1-1v-3a8 8 0 0 0-16 0v3c0 .6.4 1 1 1h2c.6 0 1-.4 1-1v-3l2-4-3-2" />
  <path d="M17 10h3a2 2 0 0 1 0 4H2" />
```